### PR TITLE
[SPARKTA-388] fix running policies number on local mode

### PR DESCRIPTION
**Pull request by acemilio**
Check when Sparta runs in **local/standalone** mode and avoid multiple policies run at the same time.

***given*** an instance of Sparta (local/standalone mode) with a started policy
***when*** you try to run other policy
***then*** return an error

***given*** an instance of Sparta (local/standalone mode) with a started policy named "A"
***when*** you try to commit a new policy named "B"
***then*** the new policy will be created but not launched

***given*** an instance of Sparta (cluster mode) with a started policy named "A"
***when*** you try to run other policy "B"
***then*** start policy "B" at the same time as "A"

### CODE REVIEW

File: serving-api/src/main/scala/com/stratio/sparkta/serving/api/service/http/PolicyContextHttpService.scala [hunk 1]
```diff
@@ -126,7 +126,7 @@ trait PolicyContextHttpService extends BaseHttpService {
                     createFragments(fragmentActor, outputs.toList ::: inputs.toList)
                     PolicyResult(policy.id.getOrElse(""), p.name)
                   case Failure(ex: Throwable) => throw new ServingCoreException(ErrorModel.toString(
-                    ErrorModel(ErrorModel.CodeErrorCreatingPolicy, s"Can't create policy")
+                    ErrorModel(ErrorModel.CodeErrorCreatingPolicy, ex.getMessage)
```
Comment (danielcsant) [inline serving-api/src/main/scala/com/stratio/sparkta/serving/api/service/http/PolicyContextHttpService.scala]: This error is for the user. I believe it is more useful for the user the old message. If you want you can pass the strackTrace as third argument in the constructor

**Comment by acemilio** [comment]
fixed

**Comment by danielcsant** [comment]
+1!

**Comment by anistal** [comment]
+1

**Comment by antonioalf** [comment]
:shipit:

**Comment by stratiocommit** [comment]
Comment accepted. Will treat it ASAP

**Comment by stratiocommit** [comment]
Squashing and checks in progress.

**Pull request merged**